src/setup_env.sh

```diff
@@ -48,9 +48,12 @@ fi
 
 if [ ! -d $CURRENT_DIR/lib/OpenNMT-py ]; then
   echo "Cloning OpenNMT-py"
-  git clone https://github.com/chenzimin/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
+  # git clone https://github.com/chenzimin/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
+  git clone https://github.com/OpenNMT/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
   echo "Installing requirements for OpenNMT-py"
-  pip3 install -r $CURRENT_DIR/lib/OpenNMT-py/requirements.txt
+  cd $CURRENT_DIR/lib/OpenNMT-py
+  pip install -r requirements.txt
```

Comment:
Can you use pip3 here

Reply:
I tried to use `pip3` here but then it fails in installing libs.

Reply:
It usually fails at the step of installing __`torch`__ lib so I learn from [OpenNMT-py's Dockerfile](https://github.com/OpenNMT/OpenNMT-py/blob/master/Dockerfile#L2)

Reply:
I would like to keep it consistent with the rest (always using python3 and pip3), and this file can be used without using docker. Can you build a image without clone and install OpenNMT, and try install it manually with python3 and pip3 to see the error message?

Reply:
Yes, I tried. Then `pip3 install -r ...` would always be killed automatically. I also tried to wget the .whl file of __`torch`__ and then install it individually, also failed.

Reply:
What error message is displayed when it is killed?

Reply:
```
Collecting torch>=1.1 (from -r requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/30/57/d5cceb0799c06733eefce80c395459f28970ebb9e896846ce96ab579a3f1/torch-1.2.0-cp36-cp36m-manylinux1_x86_64.whl (748.8MB)
./src/setup_env.sh: line 58:    38 Killed                  pip3 install -r requirements.txt
```

Reply:
So I tried it locally (building image without OpenNMT and run pip3 install), it works fine.

When searching for these errors, I found https://github.com/pytorch/pytorch/issues/1022 and https://stackoverflow.com/questions/43245196/pip-install-killed

Reply:
Hey @chenzimin 
Yes, you are right. But for my work of integrating SequenceR into Repairnator, it is better to have everything within one docker image. I just need to invoke SequenceR and get correct results. I totally agree that it is bad practice to use __Python 2__ in 2019, but from another perspective, we are just following steps from OpenNMT-py's Dockerfile, and it is more reliable to keep consistent with upstreams.

We can add more content to `README.md` if necessary.

Reply:
I am not saying using python2 is bad here, but merely that it seems to be an out of memory issue instead.

Reply:
Can you try https://stackoverflow.com/questions/44533319/how-to-assign-more-memory-to-docker-container/44533437#44533437, or just forked this project for your work? You can still sync to this project by https://help.github.com/en/articles/syncing-a-fork

Reply:
If I am using your way to build image, it works. But I can not get correct results by invoking SequenceR image.

Reply:
I know it is one OOM issue.

Reply:
Or maybe you could release one official SequenceR image to docker hub, then we do not have to make these modifications on this repo.

Reply:
> Can you try https://stackoverflow.com/questions/44533319/how-to-assign-more-memory-to-docker-container/44533437#44533437, or just forked this project for your work? You can still sync to this project by https://help.github.com/en/articles/syncing-a-fork

Yes, I increased memory size by this way and it did works. What I mean is that, I still meet other issues so that I can not utilize SequenceR normally only by invoking its image. In order to build one effective image, I have to make such changes to these files.

Reply:
Let's us discuss in the github issue instead